mds/USAGE.md

```diff
@@ -59,7 +59,7 @@ __Important notes__
 1. A JAX-RS application class with no `javax.ws.rs.ApplicationPath` annotation will not be registered.
 1. Avoid setting the JAX-RS application base URI to simply `/` to prevent URI conflicts, as explained in item 1.
 1. Property `resteasy.jaxrs.app` has been deprecated and replaced by `resteasy.jaxrs.app.classes` since version *2.2.0-RELEASE* (see [issue 35](https://github.com/paypal/resteasy-spring-boot/issues/35)). Property `resteasy.jaxrs.app` is going to be finally removed in version *3.0.0-RELEASE*.
-1. Starting on version 3.0.0, the behavior of the `scanning` JAX-RS Application subclass registration method has changed and has become more restrictive. Instead of scanning the whole classpath, it will scan only packages registered to be scanned by Spring framework (regardless of the JAX-RS Application subclass being a Spring bean or not). The reason is to improve application startup performance and avoiding classpath scanning issues in tests.
+1. Starting with version 2.1.0.Final, the behavior of the `scanning` JAX-RS Application subclass registration method has changed and has become more restrictive. Instead of scanning the whole classpath, it will scan only packages registered to be scanned by Spring framework (regardless of the JAX-RS Application subclass being a Spring bean or not). The reason is to improve application startup performance and avoiding classpath scanning issues in tests.
```

Comment:
You are breaking semantic versioning by doing so. That is fine by the way, if that is what you want, but generally that is not a good idea, and I wouldn't recommend that. If you want to introduce a backward incompatible change, do so in a major version (and that is why I planned to do that in version 3.0.0 by the way, which I was reserving for Spring Boot 2, RESTEasy 4).

Reply:
@fabiocarvalho777 I personally don't mind either way. Whatever you and @asoldano think is best, I'll obviously roll with :)

Reply:
I will also follow up with another PR to bump the Spring Boot version to the latest SB 2.1.x after this one

Reply:
@geoand @fabiocarvalho777 I've actually been in doubt about the version, as besides this and the upgrade to Spring Boot 2.1 we won't have much in the new major and I believe @geoand will soon need a release. Having said this, the fact that @fabiocarvalho777 pointed out is enough to tell us that someone might be relying on the assumption that non backward compatible changes are only coming in majors, so if you don't mind @geoand please go for 3.0.0-SNAPSHOT. Thanks